### PR TITLE
Part1

### PR DESCRIPTION
**Pull request by dnbaumann**
I think this is "done"

### CODE REVIEW

File: main.cpp [hunk 1]
```diff
@@ -58,18 +56,67 @@ video: Chapter 2 - Part 3
...
+        ignoreUnused(middleInitial); //passing each variable declared to the ignoreUnused() function
+
+    char attending = 84; // declaration of middle initial as a char variable between -128 - 127 or 0 to 255
+    
+        ignoreUnused(attending); //passing each variable declared to the ignoreUnused() function
+
+    char boy = 78; // declaration of middle initial as a char variable between -128 - 127 or 0 to 255
+
+    
+        ignoreUnused(boy); //
```
Comment (matkatmusic) [inline main.cpp:107]: FIXME:
```
b) at the end of the function, call ignoreUnused once and pass all of your variables to it.
```
FIXME:
- initialize `char` types by writing a single alpha-numeric wrapped in Single Quotes.
FIXME:
- all code inside `{ }` should have the same indentation.
- [ ] revise

File: main.cpp [hunk 2]
```diff
@@ -79,43 +126,87 @@ bool rentACar(int rentalDuration, int carType = 0)  //function declaration with
...
+    ignoreUnused(daysInMonth, monthlyRent, leaseCurrent, accessible); //passing each function parameter to the ignoreUnused() function
+    return {}; //if your function returns something other than void, add 'return {};' at the end of it.
+} 
 
 /*
  4)
- */
+ */
+ double speed(float airSpeed, float wind, float planeSpeed, int planeType = 45)  //function declaration with random number of arguments, arbitrary number of arguments have default value
+{
```
Comment (matkatmusic) [inline main.cpp:157]: Function declarations need to line up with the left edge of the text editor
- [ ] revise

File: main.cpp [hunk 3]
```diff
@@ -79,43 +126,87 @@ bool rentACar(int rentalDuration, int carType = 0)  //function declaration with
...
 /*
  5)
- */
-
+ */
+char messageFormat(bool print = true, char paperType = 25) 
+ {
+    ignoreUnused(print, paperType); //passing each function parameter to the ignoreUnused() function
+    return {}; //if your function returns something other than void, add 'return {};' at the end of it.
+ }
```
Comment (matkatmusic) [inline main.cpp:169]: Curly braces need to line up with the left edge of the start of the function declaration
- [ ] revise

File: main.cpp [hunk 4]
```diff
@@ -79,43 +126,87 @@ bool rentACar(int rentalDuration, int carType = 0)  //function declaration with
...
 /*
  6)
- */
-
+ */
+ bool tooHeavyToCarry(int objectWeight, int abilityToCarry, bool notLongerThan = true)  //function declaration with random number of arguments, arbitrary number of arguments have default value
+{
```
Comment (matkatmusic) [inline main.cpp:174]: Function declarations need to line up with the left edge of the text editor
- [ ] revise

File: main.cpp [hunk 5]
```diff
@@ -79,43 +126,87 @@ bool rentACar(int rentalDuration, int carType = 0)  //function declaration with
...
+    ignoreUnused(tooHeavyToCarry, objectWeight, abilityToCarry, notLongerThan); //passing each function parameter to the ignoreUnused() function
+    return {}; //if your function returns something other than void, add 'return {};' at the end of it.
+} 
 /*
  7)
- */
-
+ */
+ int winningHorse(int horseNumber, int raceNumber, int handicap, bool mud = false)  //function declaration with random number of arguments, arbitrary number of arguments have default value
+{
```
Comment (matkatmusic) [inline main.cpp:182]: Function declarations need to line up with the left edge of the text editor
- [ ] revise

File: main.cpp [hunk 6]
```diff
@@ -79,43 +126,87 @@ bool rentACar(int rentalDuration, int carType = 0)  //function declaration with
...
+    ignoreUnused(controlID, dawID, screenResolutionType); //passing each function parameter to the ignoreUnused() function
+    return {}; //if your function returns something other than void, add 'return {};' at the end of it.
+} 
 /*
  9)
- */
-
+ */
+ int yearOfDelivery(int yearMade, int currentDate, bool carCompleted = false)  //function declaration with random number of arguments, arbitrary number of arguments have default value
+{
```
Comment (matkatmusic) [inline main.cpp:198]: Function declarations need to line up with the left edge of the text editor
- [ ] revise

File: main.cpp [hunk 7]
```diff
@@ -135,28 +226,37 @@ int main()
...
-    
-    //10)
+    //9)
+    auto readyToDeliver = yearOfDelivery(2022, 2023, false);  
     
+    //10)
+    auto willBeReady = timeToComplete(2022, 03, 24, false); 
     
-    ignoreUnused(carRented);
+    ignoreUnused(carRented,teacherEval,lastYearProfit,nextMonthRent,actualSpeed, formattedMessage,dontCarry,winner, canBeDragged,readyToDeliver,willBeReady);
```
Comment (matkatmusic) [inline main.cpp:259]: Add 1 space after `,` to make the the code more readable
- [ ] revise

**Review by matkatmusic — CHANGES_REQUESTED** [review]
Your function names do not conform to this mantra:
- Name objects what they are
- name functions what they do.
Example:
 ```
auto lastYearProfit = profit(12.99, 2.99,10.0);
```
`profit` is not an action.  it is a noun.
`computeProfit` is an action.  it has a verb in the name.

It's important to establish this habit of using great names for your functions now before you get further into the course.

File: main.cpp (hunk 1 above)
Comment (dnbaumann) [reply main.cpp:107]: Great comments.  will fix right away.  I went back and forth on whether to write just one line or many.  If you're bored, I can show why I felt the balance of your instructions led to the way I did it as I already coded it the way you requested and changed it back (i.e., one or many ignoreUnused in the first part).

File: main.cpp (hunk 7 above)
Comment (dnbaumann) [reply main.cpp]: Did my best to resolve all errors and address all comments.  I'll admit I'm really struggling.  Not with the concepts, but I have to learn to adapt to my handicap.  My left-hand shakes badly and I have wiped out hours of work in more ways than I can count.  Just one twitch while a large block of text is highlighted and it's gone if you don't notice.  I'll get better, and i need to get used to these random twitches.  I'm going to start saving copies to an external editor so it's harder for me to clobber work.  In all events, I'm sure I still have something wrong in here.  Your assistance will be appreciated.

Comment (dnbaumann) [reply main.cpp]: I believe everything is resolved and it compiles.  Your comments are welcome as always.

**Review by matkatmusic — APPROVED** [review]

**Pull request merged**